docs/Color-Picker/Color-Picker.md

```diff
@@ -1 +1,24 @@
-TODO
\ No newline at end of file
+## Requirements
+
+### Part 1: App-Only
+* The audience will listen to the orchestra play four pieces of music.
+* Each audience member will use the app to pick from six available colors (RGB, CMY).
+* It's possible that we will want audience members of the final rehearsal to be able to choose emotions instead.
+* The audience member can pick a new color at any time, but the app will give a gentle reminder every 20 seconds.
+* During the intermission (after the four songs), each audience member's app will display both their own choices as well as the overall audience distribution. Let's call this the "results" screen.
+* The app will move from one song to the next as the event progresses without user input
+
+### Part 2: App + Central Visualization
+* In the third section, the audience will listen to the orchestra play the Coriolan overture.
+* The audience will be randomly split into two groups.
+    * In group A, the functionality is the same as the app-only section.
+    * In group B, the audience member will instead be picking emotions: Joy, sadness, anger, fear, [disgust].
```

Comment:
Not sure about disgust, might possibly need other secondary emotions. Need to finalize with Arseny.

How should we display the emotions? Text? Emojis?

Reply:
Psychology Today has a graphic showing a wheel of emotions. Not sure how accurate its structure is, but having all emotions in one place to think about might be useful. [What are Basic Emotions](https://www.psychologytoday.com/us/blog/hide-and-seek/201601/what-are-basic-emotions) 

Maybe a combination of emojis and text labels for them would allow quick identification of each option while maintaining clarity? For example: [Emoji Emotions](https://i.pinimg.com/originals/09/5a/ea/095aea060c73286d48b37a2c4dd63e56.jpg)

Reply:
I like the idea of a grid of emotions, with emojis and text. If we have a color wheel, displaying emotions in a wheel as well might bias someone to choose the emotion that's in the same position in the wheel as the color that they chose on their first listen.

Reply:
I agree. I could see how that could present a bias issue.

Reply:
Maybe overthinking this, but if there will be text in the app (like for emoji labels here), do we need to care about supporting more languages than just English?

Reply:
Great point, Jackson. Quite possibly. My understanding is that French is the primary language in Lausanne. Something to check in with Arseny about.